lib/v2/huangz/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'huangz.me': {
+        _name: 'huangz/blog',
+        blog: [
+            {
+                title: 'huangz/blog',
```

Comment:
```suggestion
                title: '文章',
```
`title` should match H3 heading of the docs